connect-inject/endpoints_controller.go

```diff
@@ -225,7 +226,25 @@ func (r *EndpointsController) createServiceRegistrations(pod corev1.Pod, service
 	proxyConfig := &api.AgentServiceConnectProxyConfig{
 		DestinationServiceName: serviceName,
 		DestinationServiceID:   serviceID,
-		Config:                 nil, // TODO: add config for metrics (upcoming PR)
+		Config:                 map[string]interface{}{},
```

Comment:
nit: I think we're to use `make(map[string]interface{})` for instantiations of empty maps.